openreview/workflows/workflows.py

```diff
@@ -337,7 +353,13 @@ def set_conference_review_request(self):
             process=self.get_process_content('workflow_process/support_process.py')
         )
 
-        self.post_invitation_edit(invitation)
+        self.client.post_invitation_edit(invitations=self.support_meta_invitation_id,
```

Comment:
@melisabok I am using the Support meta invitation to create the request form. Should I also use this invitation for the deployment/comment invitations?